### ISSUE TITLE
Should SVGs have the screen/tablet area in them?

### ISSUE THREAD

**Issue by redstrate** [linuxwacom/libwacom#832]
I notice that some SVGs have what appears to be the tablet area or screen as a rect, and they don't seem to have an id on them either. Is this intentional, and is this something we should start adding to SVGs?

**Comment by whot** [comment]
yeah, it's... intended. there's a bit of a problem with the SVGs in that for years gnome was the only user of them so there's a fair bit of cross-pollination on what they should look like.
OTOH having each project reproduce the SVGs is not smart either so we're stuck with what they have. IMO adding the outlines with a good ID would be the right thing to do but it's a fair bit of effort. I'd merge patches that do this though.

This somewhat goes double because the long-term plans for GNOME are to switch away from the current OSD anyway so an SVG with outline would also be more useful there.
